packages/integrations/src/test-utils/index.ts

```diff
@@ -0,0 +1,65 @@
+import test, { describe, beforeEach } from 'node:test';
+import * as assert from 'node:assert';
+import { Test, TestBed } from './test-bed';
+
+describe('example', () => {
+  let module: TestBed;
+  const mockStore = Test.createMockStore();
```

Comment:
I'd recommend instantiating this in `beforeEach` - that way the call count will be reset on each test, rather than relying on the tests being run in a specific order.

Reply:
Yes of course.
I just want to demonstrate the functions that I want to implement as part as the test utils.

Are those testing abilities looks legit?